### PR TITLE
feat: fund on testnet

### PR DESCRIPTION
**Pull request by neilcampbell**
- Completes #180

testnet_funding

### CODE REVIEW

File: src/features/forms/components/number-form-item.tsx [hunk 1]
```diff
@@ -42,15 +42,17 @@ export interface NumberFormItemProps<TSchema extends Record<string, unknown> = R
   decimalScale?: number
   thousandSeparator?: boolean
   placeholder?: string
+  helpText?: string | ReactElement
```
Comment (neilcampbell) [inline src/features/forms/components/number-form-item.tsx:45]: FYI. I haven't propagated helpText to all the form items, as I was trying to constrain the number of components I touched. I can do this or we can do it as we use it?

File: src/features/forms/components/validation-error-or-help-message.tsx
Comment (PatrickDinh) [inline src/features/forms/components/validation-error-or-help-message.tsx:1]: I think we should split this into 2 components, there will be scenarios when we want to display error message and help text.

File: src/features/fund/fund-page.test.tsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+      return executeComponentTest(
+        () =>
+          render(
+            <Auth0Provider domain="test" clientId="test">
+              <FundPage />
+            </Auth0Provider>
+          ),
+        async (component) => {
+          expect(1).toBe(1)
```
Comment (PatrickDinh) [inline src/features/fund/fund-page.test.tsx:106]: do we need this?

File: .github/actions/static-site-transforms/action.yaml
[the remark this answers is not in the document]
Comment (neilcampbell) [reply .github/actions/static-site-transforms/action.yaml:1]: 100%, it was lifted from there. For some reason I was thinking that only workflows are reuse-able. Will update.

File: src/features/common/components/render-inline-async-atom.tsx [hunk 3]
```diff
@@ -8,5 +8,12 @@ const transformError = (error: Error) => {
 }
 
 export function RenderInlineAsyncAtom<T>(props: RenderAsyncAtomProps<T>) {
-  return <RenderAsyncAtom transformError={transformError} fallback={<Ellipsis className="size-4 h-full animate-pulse" />} {...props} />
+  const { fallback, ...rest } = props
```
[the remark this answers is not in the document]
Comment (neilcampbell) [reply src/features/common/components/render-inline-async-atom.tsx:11]: That's fair, I'll change to now allow fallback to be passed at all then.

File: src/features/forms/components/validation-error-or-help-message.tsx
Comment (neilcampbell) [reply src/features/forms/components/validation-error-or-help-message.tsx:1]: We can. I went back and forth, but I think we actually want to display help text or error message, which is why I set up like this. If we change to a different behaviour we can split.

**Review by PatrickDinh — APPROVED** [review]

**Pull request merged**